realm/realm-library/src/androidTest/kotlin/io/realm/DictionaryTester.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm
+
+/**
+ * Contains all the methods to test in RealmDictionary plus some other convenience ones.
+ */
+interface DictionaryTester {
```

Comment:
I give that it is on a tester interface, but the names are a bit bean-ish, so you get a bit confused when reading the implementations. Probably ok to keep.